config/default.yml

```
@@ -753,6 +753,14 @@ Rails/UnknownEnv:
     - test
     - production
 
+Rails/UseTimeZone:
+  Description: 'Prefer the usage of `Time.use_zone` instead of manually updating `Time.zone` value.'
+  Enabled: true
```

Can you add the `Reference`?

```suggestion
  Description: 'Prefer the usage of `Time.use_zone` instead of manually updating `Time.zone` value.'
  Reference: https://thoughtbot.com/blog/its-about-time-zones
  Enabled: true
```